packages/dapp-high-roller/src/components/app-game/app-game.tsx

```diff
@@ -130,15 +131,16 @@ export class AppGame {
   @Element() private el: HTMLStencilElement = {} as HTMLStencilElement;
   @Prop() history: RouterHistory = {} as RouterHistory;
 
-  @Prop({ mutable: true }) myName: string = "Facundo";
   @Prop({ mutable: true }) betAmount: string = "3 ETH";
-  @Prop({ mutable: true }) opponentName: string = "John";
   @Prop({ mutable: true }) isProposing: boolean = false;
   @Prop({ mutable: true }) appInstanceId: string = "";
 
   @Prop() cfProvider: cf.Provider = {} as cf.Provider;
   @Prop({ mutable: true }) appInstance: AppInstance = {} as AppInstance;
 
+  @Prop() user: any = { username: "Facundo" };
```

Comment:
@Alonski Should probably have empty strings?

Reply:
But if the state isn't injected? Maybe these should be generic like "Bob" and "Alice"?